javascript/code-challenges/challenges-02.test.js

```diff
@@ -0,0 +1,328 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+
+CHALLENGE 1 - Review
+
+Write a function named raisedToTheThird that takes in an array of numbers and returns a new array of each of those numbers raised to the 3rd power (hint: look up Math.pow()). Use forEach to solve this problem.
+
+------------------------------------------------------------------------------------------------ */
+
+const raisedToTheThird = (arr) => {
+  let newArr = [];
+  arr.forEach(num => newArr.push(Math.pow(num,3)));
+
+  return newArr;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named addOne that, given an array of numbers, uses map to return a new array with each value simply incremented by 1. 
+------------------------------------------------------------------------------------------------ */
+
+const addOne = (arr) => {
+  let nums = arr.map( function(n, i){
+    return n + 1;
+  });
+
+  return nums;
+  // Solution code here...
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 3
+
+Write a function named addQuestion that, given an array of strings, uses map to return a new array containing each string followed by a question mark character.
+------------------------------------------------------------------------------------------------ */
+
+const addQuestion = (arr) => {
+  let question = arr.map( function(i){
+    return i +'?';
+  });
+
+  return question;
+  // Solution code here...
```

Comment:
Hey there!

Nice work on these challenges 💯 

Some feedback here: 

Try and remove and any unused code/ comments in each challenge/your projects. As time goes on and things get quite complex, it can really help improve readability and keep your code lookin' nice and clean. 

Here for ex, no need to keep the `//Solution code here...` in this code block